### PR TITLE
changes for database initialisation and unit test 

### PR DESCRIPTION
**Pull request by RohanDudeja**

### CODE REVIEW

**Comment by RohanDudeja** [comment]
Right Now changes are just for database initialisation
Will add one unit test for ref

File: test/mysql/mysql.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package mysql
```
Comment (mukesh0513) [inline test/mysql/mysql.go]: mock.go?

File: config/database.go [hunk 2]
```diff
@@ -2,22 +2,10 @@ package config
...
-	"github.com/jinzhu/gorm"
 	_ "github.com/kelseyhightower/envconfig"
-	"gopkg.in/yaml.v2"
-	"os"
 )
 
-var DB *gorm.DB
-
-// Config represents configuration
-type Config struct {
```
Comment (mukesh0513) [inline config/database.go:16]: Where is this struct moved?

Comment (RohanDudeja) [reply config/database.go:16]: moved to config/config.go

**Review by mukesh0513 — APPROVED** [review]
LGTM

**Pull request merged**